app/build.gradle

```diff
@@ -2,16 +2,18 @@ plugins {
     id 'com.android.application'
     id 'kotlin-android'
     id 'kotlin-kapt'
-    id 'com.google.secrets_gradle_plugin' version '0.4'
+    id 'com.google.android.libraries.mapsplatform.secrets-gradle-plugin'
 }
 
 android {
-    compileSdk 31
-
+    namespace "com.example.samplekotlin"
+    compileSdk 33
+    buildToolsVersion "33.0.0"
+    
     defaultConfig {
         applicationId "com.example.samplekotlin"
         minSdk 16
-        targetSdk 31
+        targetSdk 33
         versionCode 1
         versionName "1.0"
         ndkVersion = "23.1.7779620"
```

Comment:
Is the ndkVersion the same for both targetSdk versions? I wasn't sure if that was something we suggested.

Reply:
AFAIK, the NDK is only relevant if you use native C/C++ code. They added this parameter to have more precision when specifying the development environment (I guess many issues happened because of different compiler or libs versions colliding in different machines)

I updated it while doing tests but it seems to have zero impact. I did not want to remove it just to keep changes minimal, to avoid confusion about what affects what.

Reply:
Okay. Sounds good. I know that looking at the versions of NDK, this one is outdated and wanted to make sure we were good.